.editorconfig

```diff
@@ -11,6 +11,9 @@ root = true
 indent_style = space
 indent_size = 2
 
+[*.js]
+end_of_line = lf
```

Comment:
Some of my files have been using `CrLf`. The setting in `.gitattributes` doesn't cover new files so I think VS Code's been using the system default which this will fix.